src/deriver/tom/embeddings.py

```
@@ -0,0 +1,237 @@
+from typing import List, Dict, Set, Tuple
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import

(F401)